package.json

```diff
@@ -46,11 +46,13 @@
   },
 
   "scripts": {
-	"test": "mocha tests.coffee --compilers coffee:coffee-script/register"
+    "test": "mocha test/dice_test.coffee --compilers coffee:coffee-script/register"
```

Comment:
Hey I just tested the command with just `test/*` and it worked. Although now I think it should be '
'tests/*`. Plurality makes more sense when there are many test files.